src/Main.java

```diff
@@ -1,5 +1,135 @@
+import javax.swing.*;
+import java.io.*;
+import java.sql.*;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
 public class Main {
+    // Connect to your database
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        Map<String, String> env = System.getenv();
+        String endpoint = env.get("db_connection");
+        String firstName = "";
+        String lastName = "";
+        Map<Character, Integer> initialCount = new HashMap<>();
+
+        String connectionUrl =
+                "jdbc:sqlserver://" + endpoint + ";"
+                        + "database=Attias;"
+                        + "user=MCON364;"
```

Comment:
anything sensitive belongs in secrets or env variable...especially database, username adn password!